sdk/sdktypes/connection.go

```diff
@@ -28,10 +28,16 @@ func (ConnectionTraits) Validate(m *ConnectionPB) error {
 }
 
 func (ConnectionTraits) StrictValidate(m *ConnectionPB) error {
+	var errProjectID, errIntegrationID error
+	if m.Name != SchedulerConnectionName {
+		errProjectID = mandatory("project_id", m.ProjectId)
```

Comment:
i think project id is always needed?

Reply:
why? in DB we have projectID is optional. moreover cron connection isn't for a project but global one